mthree/mitigation.py

```diff
@@ -306,8 +306,8 @@ def _grab_additional_cals(self, qubits, shots=None, method='balanced', rep_delay
                                                  initial_reset=initial_reset))
             trans_qcs = transpile(circs, self.system, optimization_level=0)
 
-        # This BaseBackend check is here for Qiskit direct access.  Should be removed later.
-        if isinstance(self.system, BaseBackend):
+        # This Backend check is here for Qiskit direct access.  Should be removed later.
+        if isinstance(self.system, Backend):
```

Comment:
I think I am going to look for the class name in Qiskit direct to bypass this for now, namely `backend.__class__.__name__ == 'DirectBackend'`